### DISCUSSION TITLE
Batch modification of rows

### DISCUSSION THREAD

**Discussion by lipan1024** [Budibase/budibase#3660] [Ideas]
e.g.  I have 100 rows in my table, select 20 rows and modify them at the same time.

**Comment by mjashanks** [comment]
There is no way to do this at the moment, although I'd agree that it would be extremely useful.

Marking this as an idea...

**Reply by samg666** [reply]
If you are using custom SQL, then you can try this workaround:

- add a button on repeater row
- add button action, when the button click then store the row id to app state `SelectedIDs`, here you will need to write either Javascript or Handlebar codes, make the app state `SelectedIDs` something like array `1001,1002,1003`.
- Now you have the `SelectedIDs`, then you can batch update by custom SQL something like `update table1 set field1='value' where id in '{{SelectedIDs}}'`

**Comment by tmdoit** [comment]
Also batch insert and delete will be very useful.
